README.md

```diff
@@ -33,16 +33,9 @@ Before installing qpc on a system, review the following guidelines about install
 The Python packages that are required for running qpc on a system can be found in the `dev-requirements.txt` file. The Python packages that are required to build and test qpc from source can be found in the `requirements.txt` and `dev-requirements.txt` files.
```

Comment:
I didn't see those, should this change to poetry and the related pyproject.toml file ?

Reply:
Thanks for pointing this out. This needs to be updated to point to poetry lock.